### PR TITLE
Add unit test for EncryptOrderByItemTokenGenerator

### LINKED ISSUE #15822: Add unit test for EncryptOrderByItemTokenGenerator
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

We need to add some unit test for EncryptOrderByItemTokenGenerator.

- EncryptOrderByItemTokenGenerator#isGenerateSQLToken
- EncryptOrderByItemTokenGenerator#generateSQLTokens

### PR DESCRIPTION
**Pull request by Trydamere**
Fixes #15822.

Changes proposed in this pull request:
- add unit test for the generateSQLTokens method in EncryptOrderByItemTokenGenerator.

### CODE REVIEW

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/impl/EncryptOrderByItemTokenGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        EncryptTable encryptTable = mock(EncryptTable.class);
+        when(encryptRule.getCipherColumn("t_encrypt", "certificate_number")).thenReturn("cipher_certificate_number");
+        when(encryptRule.findAssistedQueryColumn("t_encrypt", "certificate_number")).thenReturn(Optional.of("assisted_certificate_number"));
+        when(encryptRule.findPlainColumn("t_encrypt", "certificate_number")).thenReturn(Optional.of("certificate_number_plain"));
+        when(encryptTable.findEncryptorName("certificate_number")).thenReturn(Optional.of("encryptor_name"));
+        when(encryptRule.findEncryptor("t_encrypt", "certificate_number")).thenReturn(Optional.of(mock(EncryptAlgorithm.class)));
+        when(encryptRule.findEncryptTable("t_encrypt")).thenReturn(Optional.of(encryptTable));
+        return encryptRule;
+    }
+
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/impl/EncryptOrderByItemTokenGeneratorTest.java:91]: Hi @Trydamere, can you remove this useless blank line?

**Review by strongduanmu — CHANGES_REQUESTED** [review]
Hi @Trydamere, can you sync latest master branch to solve ci error?

**Comment by Trydamere** [comment]
I will do it.

**Review by strongduanmu — APPROVED** [review]
@Trydamere Thank you.

**Pull request merged**